@xen-orchestra/proxy/src/main/mixins/remotes.js

```diff
@@ -0,0 +1,41 @@
+import { getHandler } from '@xen-orchestra/fs'
+
+export default class Remotes {
+  constructor(app, { remoteOptions }) {
+    this._remoteOptions = remoteOptions
+
+    app.api.addMethods({
+      remote: {
```

Comment:
Why not use a class for it? IMHO, it will be easy to read.

Reply:
I don't see how, feel free to show me an example.

Reply:
I would say a module for the API because it doesn't make sense to create a class for the API. 

The architecture will be:
```
 mixins
 └─ remotes
      ├─ index.js
      └─ api.js
```

- The API file will be structured like XO APIs.
- **The implementation questions and limitations**: Where we'll store the API name ? Can it be based on the mixins folder name (singular name)?

The advantage of this architecture:
- Intuitive
- If we want to add an API, we'll not take time to understand how `addMethods` works.
- Separation between the interfacing and the remotes implementation because it can have a lot of method which are not provided by the API.

Reply:
Thanks for your proposal, it's interesting.

I suggest we keep this simple (but functional) for now and we'll improve when necessary, what do you say?

Reply:
I agree :+1: